### PR TITLE
add exercise 16

### PR DESCRIPTION
**Pull request by barrantes-sofiaanne**
Please review my file thank you

### CODE REVIEW

File: barrantes/book_journal.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Book Details</title>
+</head>
+<body>
+    <h1>Book Details</h1>
+    <h2>Create Book Entry</h2>
+    <form id="createForm">
```
Comment (ElizaAlimasa) [inline barrantes/book_journal.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
    <form id="create_form">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: barrantes/book_journal.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <input type="text" id="Update_Book_Genre" name="Update_Book_Genre"><br><br>
+        
+        <label for="Update_Release_Date">Updated Release Date:</label>
+        <input type="date" id="Update_Release_Date" name="Update_Release_Date"><br><br>
+        
+        <label for="Update_Author">Updated Author:</label>
+        <input type="text" id="Update_Author" name="Update_Author"><br><br>
+        
+        <label for="Update_Rating">Updated Rating:</label>
+        <input type="number" id="Update_Rating" name="Update_Rating"><br><br>
```
Comment (ElizaAlimasa) [inline barrantes/book_journal.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

**Review by ElizaAlimasa — COMMENTED** [review]
Please view my suggested changes in your files, also update your css and js files. Other than that, Everything looks good to me.

**Review by ElizaAlimasa — APPROVED** [review]
You can now merge it

**Comment by barrantes-sofiaanne** [comment]
thank you

**Pull request merged**